packages/sources/css/src/components/selection-controls/toggle/src/index.css

```diff
@@ -118,7 +119,6 @@
   font-family: var(--vtmn-typo_font-family);
   color: var(--vtmn-semantic-color_content-primary);
   font-size: var(--vtmn-typo_text-2-font-size);
-  margin-inline-start: rem(12px);
```

Comment:
**question**: I removed this one since it seemed to me it was the same as the medium, but maybe I'm missing a point here!